base/nvti.c

```diff
@@ -169,6 +169,9 @@ typedef struct nvti
   gchar *affected; /**< @brief Affected systems */
   gchar *impact;   /**< @brief Impact of vulnerability */
 
+  gint creation_time;     /**< @brief Time of creation, seconds since epoche */
```

Comment:
I think these should be time_t instead of gint.  That's what we use in gvmd, and it matches the return from C's time().

Reply:
Done.